lib/resources/example_json/r4_examples_condition.rb

```diff
@@ -103,5 +103,74 @@ module Resources
         }
       ]
     }
+
+    R4_CONDITION_PROBLEM_UPDATE ||= {
+      "resourceType": "Condition",
+      "id": "p10070301",
+      "subject": {
+          "reference": "Patient/1316024"
+      },
+      "code": {
+          "coding": [
+              {
+                  "system": "http://hl7.org/fhir/sid/icd-10-cm",
+                  "code": "T86.03",
+                  "display": "Bone marrow transplant infection",
+                  "userSelected": true
+              }
+          ],
+          "text": "Bone marrow transplant infection"
+      },
+      "category": [
+          {
+              "coding": [
+                  {
+                      "system": "http://terminology.hl7.org/CodeSystem/condition-category",
+                      "code": "problem-list-item"
+                  }
+              ],
+              "text": "Problem"
+          }
+      ],
+      "clinicalStatus": {
+          "coding": [
+              {
+                  "system": "http://terminology.hl7.org/CodeSystem/condition-clinical",
+                  "code": "resolved"
+              }
+          ]
+      },
+      "verificationStatus": {
+          "coding": [
+              {
+                  "system": "http://terminology.hl7.org/CodeSystem/condition-ver-status",
+                  "code": "differential"
+              }
+          ]
+      },
+      "severity": {
+          "coding": [
+              {
+                  "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/12022",
+                  "code": "3294",
+                  "display": "Moderate",
+                  "userSelected": true
+              }
+          ],
+          "text": "Moderate"
+      },
+      "abatementDateTime": "2010-11-08T02:00:00.00+00:00",
```

Comment:
let's swtich these to just dates, since that's what we return.

Reply:
Updated [here](https://github.com/nishanthnamana/fhir.cerner.com/commit/91fee55a512ed5f620c71861b3940e8479c3f61a#diff-c6a39eaf3cd21cf52bf6cce80702b4d8R162).